geometry_msgs/Point start
geometry_msgs/Point goal
geometry_msgs/Twist twist
geometry_msgs/Twist acceleration
# mavbench_msgs/multiDOFpoint start
# mavbench_msgs/multiDOFpoint end
int32 length
int32 width
int32 n_pts_per_dir
Header header
int32 call_func
---
# trajectory_msgs/MultiDOFJointTrajectory multiDOFtrajectory
mavbench_msgs/multiDOFtrajectory multiDOFtrajectory
int32 unknown
bool path_found


